pkg/helm/utils.go

```diff
@@ -45,6 +46,52 @@ func CreateTensorleapChartValues(useGpu bool, dataDir string, disableMetrics boo
 	}
 }
 
+func CreateTensorleapChartValuesFormOldValues(oldValues Record) (Record, error) {
+
+	disableMetrics := false
+	useGpu := false
+
+	errFailedGettingOldValue := fmt.Errorf("failed getting old values")
+
+	engineVal, ok := oldValues["tensorleap-engine"]
+	if !ok {
+		return nil, errFailedGettingOldValue
+	}
+	engineValMap, ok := engineVal.(map[string]interface{})
+	if !ok {
+		return nil, errFailedGettingOldValue
+	}
+	useGpuVal, ok := engineValMap["gpu"]
+	if ok {
+		useGpu = useGpuVal.(bool)
+	}
+	dataDirVal, ok := engineValMap["localDataDirectory"]
+	if !ok {
+		return nil, errFailedGettingOldValue
+	}
+	dataDir, ok := dataDirVal.(string)
+	if !ok {
+		return nil, errFailedGettingOldValue
+	}
+
+	nodeServerVal, ok := oldValues["tensorleap-node-server"]
```

Comment:
Think on agnostic way to do that

Reply:
Sure we need to go template to the helm chart lets open ticket